cmake/SetupCompilers.cmake

```diff
@@ -40,7 +40,9 @@
 # 
 ###############################################################################
 
-set(CMAKE_CXX_STANDARD 14)
+if (NOT RAJA_ENABLE_CLANG_CUDA)
+  set(CMAKE_CXX_STANDARD 14)
```

Comment:
This flag only enables C++14 if it's supported by the compiler. There is no code that relies on C++14. It silently falls back to the highest supported standard. The `RAJA` target is C++11, so enforces we use at least C++11